client/src/Booking.jsx

```diff
@@ -151,12 +151,13 @@ class Booking extends React.Component {
           <div className="dates-and-guests">
             <div className="row">
               <div className="col-xs-6 check-in-btn">
-                <CheckIn month={this.state.current_month} campId={this.state.campId} click={this.handleClick} submit={this.handleSubmit} />
                 <CheckIn
                   month={this.state.current_month}
                   campId={this.state.campId}
+                  inventory={this.state.inventory}
                   onClick={this.handleClick}
                   submit={this.handleSubmit}
+                  book={this.book}
```

Comment:
this section is showing up on my front end., which is awesome! nice work getting this calendar to work, I know it was pretty rough math wise for a while ;)